packages/trader/src/exchange-adapters/CreateExchangeAdapter.ts

```diff
@@ -0,0 +1,47 @@
+const assert = require("assert");
```

Comment:
This file is very similar to the `CreatePriceFeed` except it create `ExchangeAdapters`. For now, there is just one exchange adapter. when we add additional exchange support to this bot we will add them here.